tests/integration/test_store.py

```diff
@@ -148,6 +148,13 @@ def store_event():
     with pytest.raises(HTTPError):
         relay.send_event(project_id, {"message": "invalid"})
 
+    # Generated outcome has reason code 'generic':
+    outcome_envelope = mini_sentry.captured_events.get(timeout=1)
+    outcome = json.loads(outcome_envelope.items[0].payload.bytes)
+    assert outcome["rate_limited_events"] == [
+        {"reason": "generic", "category": "error", "quantity": 1}
+    ]
```

Comment:
In https://github.com/getsentry/relay/pull/1967/commits/e62d5695ed5dcfc0e07c2ea6bf2c77d8a9e4f90d, this outcome had `"reason": ""`.